library/RevAPI/Attachments.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace RevAPI;
+
+class Attachments extends \ArrayIterator
+{
+    /**
+     * @var Rev
+     */
+    protected $rev;
+
+    protected $data;
+
+    public function __construct(Rev $rev, $attachments_array) {
+        $this->rev = $rev;
+        $this->data = $attachments_array;
+        parent::__construct($attachments_array);
+    }
+
+    /**
+     * @return Attachment
+     */
+    public function current()
+    {
+        $current = parent::current();
+        return new Attachment($this->rev, $current);
```

Comment:
This may not be a huge deal, but by constructing a new object in current (without an object cache) will result in each iteration over this object returning different objects. Perhaps this is a design choice to make this quick and only force a clean re-iteration.